L1Trigger/L1THGCal/test/HGCalTriggerGeomTesterV9Imp3.cc

```diff
@@ -649,6 +653,60 @@ bool HGCalTriggerGeomTesterV9Imp3::checkMappingConsistency() {
         }
       }
     }
+
+    // Filling Stage 1 FPGA -> modules
+
+    edm::LogPrint("ModuleCheck") << "Checking module -> stage-1 -> module consistency";
```

Comment:
Would it be possible to add also checks of the Stage1 <-> Stage2 mappings?

Reply:
I have added this (but maybe could you confirm this is as you expected?)

Reply:
Thanks, that looks good